main.py

```
@@ -6,97 +6,352 @@
+    '''
+    def get_main_def():
+        CustomDEMPlugin.set_project_crs()
+        CustomDEMPlugin.enable_processing_algorithms()
+        CustomDEMPlugin.add_opentopo_layer()
+        x, y = CustomDEMPlugin.get_coordinates()
+        if x is None or y is None:
+            return
+        longitude, latitude = CustomDEMPlugin.transform_coordinates(x, y)
+        bbox = [longitude - 0.5, latitude - 0.5, longitude + 0.5, latitude + 0.5]
+        dem_path = CustomDEMPlugin.download_dem(bbox, CustomDEMPlugin.project_folder)
+        dem_layer = CustomDEMPlugin.add_dem_layer(dem_path)
```

Ruff (0.8.2)

Local variable `dem_layer` is assigned to but never used

Remove assignment to unused variable `dem_layer`

(F841)